cmd/lotus-miner/actor.go

```diff
@@ -513,7 +513,9 @@ var actorControlList = &cli.Command{
 			}
 			kstr := k.String()
 			if !cctx.Bool("verbose") {
-				kstr = kstr[:9] + "..."
+				if len(kstr) > 9 {
+					kstr = kstr[:6] + "..."
+				}
```

Comment:
In an else statement here I would print the whole string (it should fit). And then apply the same thing to `lotus-shed/actor.go`

Reply:
> In an else statement here I would print the whole string (it should fit).

I might be mistaken here, but printing the whole string here would just create an additional output of the key in the wrong place?  As `kstr` is written to the [tablewriter and flushed later on](https://github.com/filecoin-project/lotus/blob/3e74819519815cba11613146cfa92480a178c90a/cmd/lotus-miner/actor.go#L551-L566).

> And then apply the same thing to lotus-shed/actor.go

Oh, completely forgot to fix the lotus-shed cmd 😅